### ISSUE TITLE
Publish C hooks

### ISSUE THREAD

**Issue by mcandre** [pikkr/pikkr#1]
Awesome project! Could C hooks be published, so that nonRust languages can take advantage of this accelerated library?

https://doc.rust-lang.org/1.5.0/book/rust-inside-other-languages.html

**Comment by kjmrknsn** [comment]
@mcandre Thank you for your comment. I do not know how much overhead occurs when a Rust function is called from other programming languages but it looks very nice if other languages could use Pikkr.

**Comment by dongx-psu** [comment]
Agreed to this thread as well. I would be nice if pikkr can be used in C or C++.
